client/landing/jetpack-cloud/sections.js

```diff
@@ -0,0 +1,10 @@
+module.exports = [
```

Comment:
Now that this is a section in the main entry point, should we just remove this file and move the definition in the main sections file?

Reply:
:+1: